.circleci/Dockerfile

```diff
@@ -1,7 +1,7 @@
-# openjdk 11.0.9
-# node 14.15.0
+# openjdk 17.0.2
+# node 16.13.2
```

Comment:
@WebsByTodd it might be a good idea to bump the `.nvmrc` to `16.13.2` to match, I tried this but got failures that I didn't understand. Node deps could probably use a refresh too.